poollet/volumepoollet/controllers/volumepool_controller.go

```diff
@@ -116,7 +121,61 @@ func (r *VolumePoolReconciler) reconcile(ctx context.Context, log logr.Logger, v
 	return ctrl.Result{}, nil
 }
 
+func (r *VolumePoolReconciler) oriClassEventSource(mgr ctrl.Manager) (source.Source, error) {
```

Comment:
Better create a new `VolumePoolAnnotatorReconciler` (similar to `VolumeAnnotatorReconciler` and follow a similar strategy as for the `VolumeAnnotatorReconciler`.